dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/bnd.bnd

```diff
@@ -47,11 +47,19 @@ tested.features:\
 
 -buildpath: \
 	com.ibm.ws.componenttest;version=latest,\
+	io.openliberty.jakarta.servlet.6.0;version=latest,\
+	io.openliberty.jakarta.restfulWS.3.1;version=latest,\
+	io.openliberty.jakarta.cdi.4.0;version=latest,\
 	io.openliberty.microprofile.telemetry.internal_fat.common,\
 	io.openliberty.io.opentelemetry;version=latest,\
+	io.openliberty.org.eclipse.microprofile.rest.client.3.0;version=latest,\
+	io.openliberty.jakarta.annotation.2.1;version=latest,\
+	io.openliberty.jakarta.concurrency.3.0;version=latest,\
```

Comment:
Is it possible to write these tests using the EE7 APIs that the rest of the tests in this bucket depend on and let the transformer convert them to the jakarta ones?

While I think these tests should only run against EE9 and 10 and we can have both the javax and jakarta APIs on the buildpath, I'd prefer not to because it makes it easy to pull in the jakarta version of an annotation when the test needs the javax one.